docs/en/functionalities.rst

```diff
@@ -147,9 +147,9 @@ Issuance from a Touchpoint of the Authentic Source
 Below are illustrated the User Experience requirements for the issuance of an Electronic Attribute Attestation from the Catalogue that the Wallet Solution Provider MUST guarantee through their own Wallet Solution: 
 
 - The User interacts with the Engagement Button clearly displayed in the Touchpoint interface; 
-- The User selects the Wallet Solution with which to proceed, through an interface that SHOULD follow the directions and functionalities described for the *Selection Page* in the :ref:`functionalities:Authentication section`; 
-- (*cross-device only*) the User scans the QR code that invokes the opening of their chosen Wallet Instance, through an interface that SHOULD follow the directions and functionalities described for the *QR Code Page* in the :ref:`functionalities:Authentication section`; 
-- (*cross-device only*) the User displays a message inviting them to continue on their chosen Wallet Instance, through an interface that SHOULD follow the directions and functionalities described for the *Waiting Page* in the :ref:`functionalities:Authentication section`; 
+- The User selects the Wallet Solution with which to proceed, through an interface that SHOULD follow the directions and functionalities described for the *Selection Page* in the :ref:`functionalities:Authentication` section; 
+- (*cross-device only*) the User scans the QR code that invokes the opening of their chosen Wallet Instance, through an interface that SHOULD follow the directions and functionalities described for the *QR Code Page* in the :ref:`functionalities:Authentication` section; 
+- (*cross-device only*) the User displays a message inviting them to continue on their chosen Wallet Instance, through an interface that SHOULD follow the directions and functionalities described for the *Waiting Page* in the :ref:`functionalities:Authentication` section; 
```

Comment:
@steps88 Waiting Page should be more contextualized, Waiting for?

User Authentication Selection Waiting Page or something